app/src/main/res/layout/fragment_tabhome.xml

```diff
@@ -1,18 +1,191 @@
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/nowPlayingRecyclerView"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            android:layout_marginTop="10dp"
+            >
+            <TextView
+                android:id="@+id/upComingTextView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/upcoming"
+                android:textSize="@dimen/sp_20"
+                android:textStyle="bold"
+                android:textColor="@color/colorWhite"
+                android:layout_centerVertical="true"
+                android:layout_alignParentLeft="true"
+                />
+            <Button
+                android:id="@+id/moreUpComingButton"
+                android:layout_width="@dimen/dp_60"
+                android:layout_height="@dimen/dp_30"
+                android:background="@color/colorOrange"
+                android:text="@string/more"
+                android:textColor="@color/colorWhite"
+                android:textStyle="bold"
+                android:layout_alignParentRight="true"
+                />
+        </RelativeLayout>
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/upComingRecyclerView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/containerUpComing"
+            />
+
+        <RelativeLayout
+            android:id="@+id/containerTopRated"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/upComingRecyclerView"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            android:layout_marginTop="10dp"
+            >
+            <TextView
+                android:id="@+id/topRatedTextView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/top_rated"
+                android:textSize="@dimen/sp_20"
+                android:textStyle="bold"
+                android:textColor="@color/colorWhite"
+                android:layout_centerVertical="true"
+                android:layout_alignParentLeft="true"
+                />
+            <Button
+                android:id="@+id/moreTopRatedButton"
+                android:layout_width="@dimen/dp_60"
+                android:layout_height="@dimen/dp_30"
+                android:background="@color/colorOrange"
+                android:text="@string/more"
+                android:textColor="@color/colorWhite"
+                android:textStyle="bold"
+                android:layout_alignParentRight="true"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_alignParentEnd="true"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. **NOTE**: If your `minSdkVersion` is less than 17, you should add **both** the older left/right attributes **as well as** the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)